recipes/xopen/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "xopen" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "770d51ebb3b865e2ccaee05188e8a96cb959633de4456caa8d39ca52a878ca0f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - bz2file;   # [py27]
+    - pip
+    - python
+    - pytest
+    - setuptools_scm
+  run:
+    - bz2file;   # [py27]
```

Please fix this name
```suggestion
    - bz2file   # [py27]
```